modules/cache.js

```diff
@@ -404,6 +402,9 @@ Cache.prototype.onSyncFinished = function() {
 
 Cache.prototype.KEYS = {
 	transactionCount: 'transactionCount',
+	blocksApi: '/api/blocks*',
+	transactionsApi: '/api/transactions*',
+	delegatesApi: '/api/delegates*',
```

Comment:
Where this is used?

Reply:
You mean the `delegatesApi`?
If so, I meant to use it in `onFinishRound` and `onTransactionsSaved` to keep consistency but it seems I've forgotten.
Good catch! I will added it.

Reply:
Yep, thanks.